sonar-kotlin-metrics/src/main/java/org/sonarsource/kotlin/metrics/MetricVisitor.kt

```diff
@@ -153,6 +171,12 @@ private class KtMetricVisitor : KtTreeVisitorVoid() {
         super.visitBlockExpression(expression)
     }
 
+    override fun visitImportDirective(importDirective: KtImportDirective) {
+        hasAndroidImports = hasAndroidImports ||
+            ANDROID_PACKAGES.any { importDirective.importPath?.fqName?.startsWith(it) ?: false }
```

Comment:
To have full branch coverage on this line, I would need an import directive with `null` `importPath`, which is possible according to the type, but impossible according to [the grammar](https://kotlinlang.org/docs/reference/grammar.html#importList).

Attempts like `import` or `import as alias` all result in parsing failures.